java/hiro-client/src/main/java/co/arago/hiro/client/rest/DefaultWebSocketClient.java

```diff
@@ -226,7 +227,13 @@ public void onTextFrame(String payload, boolean finalFragment, int rsv) {
                         }
                     } else {
                         exitOnClose = true;
-                        process(dataListener, payload);
+                        doReconnect = false;
+                        // process(dataListener, payload);
+                        Object message = error.get("message");
+                        onError(new HiroException(
```

Comment:
this is rather reserved for ws protocol errors and not graph errors
so far graph errors was always propagated as special message over data

Reply:
So maybe propagate them to the dataListener and logListener? Or shall we just reject this pull request and leave it the way it was?

Reply:
pls do not modify error propagation.
other changes might be applied

Reply:
Ok. I rolled back to the old way but kept the renaming of the flag.